source/fem-dem/gls_vans.cc

```diff
@@ -909,9 +958,12 @@ GLSVANSSolver<dim>::assemble_local_system_rhs(
 
   for (auto &pf_assembler : particle_fluid_assemblers)
     {
-      pf_assembler->calculate_particle_fluid_interactions(scratch_data);
+      pf_assembler->calculate_particle_fluid_interactions(
+        scratch_data, this->simulation_parameters);
```

Comment:
Why does the call to calculate patricle fluid interaction requires the simulation parameter as a run time input argument? Why do you need all simulation parameters? I would give the required subset of parameters to the constructor of the assembler itself.

Reply:
done